editor/animation_track_editor.cpp

```diff
@@ -2541,6 +2541,15 @@ void AnimationTrackEditor::set_animation(const Ref<Animation> &p_anim) {
 		step->set_read_only(false);
 		snap->set_disabled(false);
 		snap_mode->set_disabled(false);
+
+		imported_anim_warning->set_visible(false);
```

Comment:
Usually we use `hide()` and `show()` for GUI logic.

Reply:
Replaced them all.